example-apps/internal-knowledge-search/requirements.txt

```diff
@@ -0,0 +1,52 @@
+#
+# This file is autogenerated by pip-compile with Python 3.11
+# by the following command:
+#
+#    pip-compile
+#
+blinker==1.6.2
+    # via flask
+build==1.0.3
+    # via pip-tools
+certifi==2023.7.22
+    # via elastic-transport
+click==8.1.7
+    # via
+    #   flask
+    #   pip-tools
```

Comment:
some of these deps might not be needed here. Not the end of the world if this is just copy-pasted from another example app, but I'd think we wouldn't need click, jinja, markup, etc.

Reply:
These are all dependencies of dependencies. Requirements.txt was auto-generated from requirements.in, which I culled.